pkg/renderer/sgml/table.go

```diff
@@ -42,21 +41,20 @@ func (r *sgmlRenderer) renderTable(ctx *renderer.Context, t types.Table) (string
 	}
 
 	if t.Attributes.Has(types.AttrTitle) {
-		number = ctx.GetAndIncrementTableCounter()
-		if s, ok := t.Attributes.GetAsString(types.AttrCaption); ok {
-			caption.WriteString(s)
-		} else {
-			err := r.tableCaption.Execute(caption, struct {
-				TableNumber int
-				Title       sanitized
-			}{
-				TableNumber: number,
-				Title:       title,
-			})
-			if err != nil {
-				return "", errors.Wrap(err, "unable to format table caption")
-			}
+		c, ok := t.Attributes.GetAsString(types.AttrCaption)
+		if !ok {
+			c, _ = ctx.Attributes.GetAsString(types.AttrTableCaption)
 		}
+
```

Comment:
should we wait for https://github.com/bytesparadise/libasciidoc/pull/715 to be merged to refactor this part, or address it later? In other words, can this PR only address the `table-caption` attribute?

Reply:
Yes, just merge this -- there are no dependencies.

Reply:
sorry, I'm not following you... The comment below says that there is a "very primitive and incomplete replacement of the counter attribute only." 
Is it something that will be refactored/improved with #715?

Reply:
Work has to occur after #715 to support attribute values that themselves contain attribute references.  (Meaning recursive attribute substitution.)  That is required -- and will be something done *after* #715 -- 715 is just enabling work to make that happen, and the approach here includes work that will be retained, but also a hack that will be replaced with real attribute substitution later.

Reply:
(Didn't mean to resolve the conversation.)

More precisely we need #604 as well as #715 before we can eliminate this hack.

Reply:
ok, I see.

Reply:
thanks for the clarification